packages/nextjs/components/scaffold-eth/RainbowKitCustomConnectButton.tsx

```diff
@@ -31,15 +34,43 @@ export const RainbowKitCustomConnectButton = () => {
 
               if (chain.unsupported || chain.id !== configuredNetwork.id) {
                 return (
-                  <>
-                    <span className="text-xs" style={{ color: networkColor }}>
-                      {configuredNetwork.name}
-                    </span>
-                    <button className="btn btn-sm btn-error ml-2" onClick={openChainModal} type="button">
+                  <div className="dropdown dropdown-end">
+                    <button tabIndex={0} className="btn btn-error btn-sm dropdown-toggle">
                       <span>Wrong network</span>
                       <ChevronDownIcon className="h-6 w-4 ml-2 sm:ml-0" />
                     </button>
-                  </>
+                    <ul tabIndex={0} className="dropdown-content menu p-2 shadow bg-base-100 rounded-box w-56">
+                      <li>
+                        <a
+                          className="menu-item"
+                          href="#"
+                          onClick={() => {
+                            if (switchNetwork) {
+                              switchNetwork(configuredNetwork.id);
+                            }
+                          }}
+                        >
+                          <ArrowsRightLeftIcon className="h-6 w-4 ml-2 sm:ml-0" />
+                          <span>
+                            Switch to <span style={{ color: networkColor }}>{configuredNetwork.name}</span>
+                          </span>
+                        </a>
+                      </li>
+                      <li>
+                        <a
+                          className="menu-item text-red-500" // Add the 'text-red-500' class
+                          href="#"
+                          onClick={() => {
+                            disconnect();
+                          }}
+                        >
+                          <ArrowLeftOnRectangleIcon className="h-6 w-4 ml-2 sm:ml-0 text-red-500" />{" "}
+                          {/* Add the 'text-red-500' class */}
+                          <span className="text-red-500">Disconnect</span> {/* Add the 'text-red-500' class */}
```

Comment:
Remove the comment, also we can just add `className="text-red-500"` to parent element and it will cascade its effects to child's so you don't have to mention `className="text-red-500"` on each child

Reply:
Done!